spec/presenters/finders/finder_signup_content_item_presenter_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+RSpec.describe FinderSignupContentItemPresenter do
+  file = 'lib/documents/schemas/cma_cases.json'
+  read_file = File.read(file)
+  let(:payload) { JSON.parse(read_file) }
+  let(:finder_signup_content_presenter) { FinderSignupContentItemPresenter.new(payload, File.mtime(file)) }
+  let(:presented_data) { finder_signup_content_presenter.to_json }
+
+  describe "#to_json" do
+    it "is valid against the content schemas" do
+      expect(presented_data[:schema_name]).to eq("finder_email_signup")
+      expect(presented_data).to be_valid_against_schema("finder_email_signup")
+    end
+  end
+end
```

Comment:
I was thinking it might be nice to iterate through all the schemas in `lib/documents/schemas` and validate them in this test. I think you could do `Dir["lib/documents/schemas/*.json"].each` or something similar.

I think this would help to future-proof us if we decide to make changes to the content schema and want to check that all of our schemas still validate. What do you think?